modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/GetEntryRenderDataMVCResourceCommand.java

```diff
@@ -183,12 +186,44 @@ else if (ctEntry.getChangeType() ==
 				ctEntry.getModelClassPK());
 
 			if (rightModel != null) {
-				boolean activeCTCollection = ParamUtil.getBoolean(
-					resourceRequest, "activeCTCollection");
-
-				if (activeCTCollection) {
-					editURL = ctDisplayRenderer.getEditURL(
-						httpServletRequest, rightModel);
+				String editURL = _ctDisplayRendererRegistry.getEditURL(
```

Comment:
@stsquared99 This part logic seems similar to the other two, also there are some hard coded string duplicated, is it possible to extract a method out rather than duplicating the code?